corehq/apps/api/tests/keyset_paginator_tests.py

```diff
@@ -0,0 +1,152 @@
+from django.test import SimpleTestCase
+from django.http import QueryDict
+from urllib.parse import urlparse
+from itertools import islice
+from tastypie.exceptions import BadRequest
+from corehq.apps.api.keyset_paginator import KeysetPaginator
+
+
+class SequenceQuery:
```

Comment:
what model is this supposed to emulate in the real code? can you add a comment there?

Reply:
The `KeysetPaginator` contains a docstring indicating:
> `objects` is expected to represent a query object that exposes an `.execute(limit)`
        method that returns an iterable, and a `get_query_params(object)` method to retrieve the parameters
        for the next query

`SequenceQuery` is a a simple version of that interface where it wraps a basic sequence. The closest class it represents is currently `IterableEnterpriseFormQuery` within the iterators module, but I don't want to make that link explicit. The tests were written against the docstring. If you provide an `objects` parameter that contains an `execute` method that returns an iterable, `KeysetPaginator` should work correctly. I'm having a hard time coming up with a more succinct term for that interface.